_posts/2017-08-25-aug-25.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "This Week In Erlang Aug 25"
+description: ""
+date: 2017-08-25
+tags: [erlang, news]
+comments: false
+share: true
+---
+
+Hello and welcome to "This week in Erlang"! As expected, there was a lot of action on the OTP side this week, and maybe a little too slow everywhere else. However, there was a piece of great news that came out this week. Read on!
+
+### Articles and Blog posts
+ * Bet365's CEO announced that they will be buying all of Basho's remaining assets and making everything open source: <http://lists.basho.com/pipermail/riak-users_lists.basho.com/2017-August/019500.html>
+
+ * There was a discussion on Hackernews about Pierre Fenoll's Supercompiler Pass project: <https://news.ycombinator.com/item?id=15064127>
+
+ * Andrew Bennett (@potatosaladx) wrote another great article this week. It is about load testing Cowboy 2.0 and how it holds against Cowboy 1.1. Highly recommend reading it: <https://potatosalad.io/2017/08/20/load-testing-cowboy-2-0-0-rc-1.html>
+ 
+### Library updates
+ * Cowboy 2.0-rc2 was released with bunch of important bug fixes: <https://ninenines.eu/articles/cowboy-2.0.0-rc.2/>
+
+ * A new version of Rebar3 was released. Hurry and update since it's a big release: <https://github.com/erlang/rebar3/releases/tag/3.4.3>
+
+
+### OTP Updates
+ * OTP 20.0.3 was released this week, with what seems like a few security fixes. Read about it at <https://groups.google.com/forum/#!topic/erlang-programming/ThszrBokews> and download the new package at <https://github.com/erlang/otp/releases/tag/OTP-20.0.3>
+
+ * There was a lot of PR merging happening on master OTP branch. But one that I particularly liked was Björn's BEAM improvement commit. Prepare for performance improvements coming in the next release: <https://github.com/erlang/otp/commit/fbf740d68600b59dc5fa7bd76d0aa0d019e81a75>
```

Comment:
I found 4 misspelled words in this sentence: 
- <https
- Björn
- OTP
	- *Suggested Replacements:* olp, ot, ptp, otc, opt, op, tp, opp, top
- *
	- *Suggested Replacements:* , k, p, u, e, z, j, o, t, d, y, i, n, s, c, x, h, m, r, b, w, g, l, q, a, v, f